xbmc/network/DNSNameCache.h

```diff
@@ -40,7 +40,7 @@ class CDNSNameCache final
    *
    * \return true if the IP is cached
    */
-  static bool GetCached(const std::string& strHostName, std::string& strIpAddress);
+  bool GetCached(const std::string& strHostName, std::string& strIpAddress);
```

Comment:
Could this member function be `const`now?

Reply:
Only if I make `m_hostToIp` `mutable`.